md/handling-documents.md

```diff
@@ -1,147 +1,146 @@
-# Handle documents
 
-This page shows an example of how to manipulate documents in a Bonita BPM process using the Java APIs. 
+# Handle documents with Java APIs
 
-A document is treated in a similar way to a variable in Bonita BPM Engine database. It is defined in Bonita BPM Studio or programmatically, and is stored in the Bonita BPM Engine database. Inside a process instance, a document is a byte stream.
+This page shows examples on how to manipulate documents in a Bonita process using the Java APIs.  
+You might want to consult the [document documentation](documents.md) first.
 
-## Configure a process document
+A document is treated in a similar way to a variable in Bonita Engine database. It is defined in Bonita Studio or programmatically, and is stored in the Bonita Engine database.
 
-To configure a document in a process definition, go to the **Details** panel, **General** view, **Documents** pane.
-An external document is specified by URI. An internal document is specified by path and file name, by clicking the **_Browse..._** button.
+## Set or update the value of a document in a process operation using a script
 
-![Configure documents](images/images-6_0/documents_declarations.png)
+In a process operation, you might want to set / update the value of a document data using a groovy script instead of simply use a contract input.  
+The script should return a _DocumentValue_ .  
+**N.B**: _All the following exemples deal with single document data. For a multiple document data, just adapt the scripts to return a list  of DocumentInput instead of a document input._  
```

Comment:
```suggestion
**N.B**: _All the following exemples deal with single document data. For multiple document data, just adapt the scripts to return a list  of DocumentInput instead of a document input._  
```